src/extensions/FileUploadExt.ts

```
@@ -0,0 +1,2205 @@
+    const parseMode = options.parseMode || 'auto';
+    switch (parseMode) {
+        case 'always':
+            return true;
+        case 'never':
+            return false;
+        case 'ask':
+            // In an actual implementation, you could show a dialog here
+            // For now, we'll assume the user said yes for parseable documents
+            return true;
+        case 'auto':
+        default:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)